docs/assets/main.css

```
@@ -0,0 +1,1957 @@
+    background-color: var(--color-red-50);
+  }
+  .bg-secondary\/5 {
+    background-color: color-mix(in oklab, var(--color-secondary) 5%, transparent);
+  }
+  .bg-gradient-to-r {
+    --tw-gradient-position: to right in oklab,;
+    background-image: linear-gradient(var(--tw-gradient-stops));
+  }
+  .bg-gradient-to-tr {
+    --tw-gradient-position: to top right in oklab,;
+    background-image: linear-gradient(var(--tw-gradient-stops));
```

Biome (1.9.4)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)